virttest/utils_net.py

```diff
@@ -3313,8 +3313,7 @@ def __verify_neigh(ip, macs, dev, timeout, session=None, **dargs):
         return bool(utils_misc.wait_for(lambda: verify_func(ip, macs, dev,
                                                             timeout,
                                                             session=session, **dargs),
-                                        timeout,
-                                        text="Retry verifying IP address"))
```

Comment:
can we add a single log before calling `wait_for()` ?

Reply:
Do you have an example?

Reply:
@luckyh, I am okay with this change. I was mentioning something like below that helps us during debugging,
```patch
diff --git a/virttest/utils_net.py b/virttest/utils_net.py
index a743bdd2..bf7b7120 100644
--- a/virttest/utils_net.py
+++ b/virttest/utils_net.py
@@ -3310,11 +3310,11 @@ def verify_ip_address_ownership(ip, macs, timeout=60.0, devs=None,
     verify_func = __verify_neigh if ip_ver == 6 else __arping
     for dev in devs:
         # VM might take some time to respond after migration
+        logging.debug("verify IP address ownership for interface: %s", dev)
         return bool(utils_misc.wait_for(lambda: verify_func(ip, macs, dev,
                                                             timeout,
                                                             session=session, **dargs),
-                                        timeout,
-                                        text="Retry verifying IP address"))
+                                        timeout))
```

Reply:
@luckyh Do you want adopt the above suggestion and remove this log this time before we have better improvement? I am testing Windows a lot recently and the message is truly annoying...

Reply:
@balamuruhans @vivianQizhu honestly, I think it is not necessary (as I already explained below), so I don't intent to add it this time, thanks.

Reply:
@luckyh I faced difficulty to debug when I worked for https://github.com/avocado-framework/avocado-vt/commit/e1d50763511aa9848c6714adf66546102b6349bb, it is not harm a single line that won't flood the log. If you still not willing, please feel free to merge it. Thanks!

Reply:
@balamuruhans in the case of calling `verify_ip_address_ownership` in the `virt_vm.get_address` context, there is already a log, `"Dropped the address cache of HWADDR ($MAC)"`, being raised when failed to verify the ownership, otherwise, callers should make the log and explain the reason. To me, as I can't see a significant help for debugging by adding such the above log, I'd like to leave it to the future and add it when we realize that we have to do that. Anyway, thanks for your understanding!